llvm/test/CodeGen/RISCV/memcmp-optsize.ll

```diff
@@ -3,6 +3,10 @@
 ; RUN:   | FileCheck %s --check-prefix=CHECK-ALIGNED-RV32
 ; RUN: sed 's/iXLen/i64/g' %s | llc -mtriple=riscv64 -O2  \
 ; RUN:   | FileCheck %s --check-prefix=CHECK-ALIGNED-RV64
+; RUN: sed 's/iXLen/i32/g' %s | llc -mtriple=riscv32 -mattr=+zbb,+zbkb -O2  \
```

Comment:
Does Zbkb provide anything over Zbb for these tests? I would like to see Zbb by itself too since it doesn't have pack. I'm not sure if the RVA23 profiles allows Zbkb as an optional extension or not.